util/threading.h

```diff
@@ -104,6 +108,7 @@ class BoundedTopology {
       enabled_lps.Set(lp);
     }
 
+#if !GEMMA_DISABLE_TOPOLOGY
     if (HWY_LIKELY(!topology_.packages.empty())) {
       InitFromTopology(enabled_lps, package_slice, cluster_slice);
     }
```

Comment:
I think we could just put an `#endif` here - without InitFromTopology, we will not have filled packages_, which is what NumPackages checks, so we can use the same fallback.